.github/workflows/backend_cd.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Build and push Docker image
         uses: docker/build-push-action@v2
         with:
-          context: .
+          context: "./apps/backend"
```

Comment:
Why is this here?

Reply:
So that the backend code is only deployed to the backend server when the backend code changes.